teams/types.toml

```
@@ -39,3 +39,7 @@ extra-people = ["eholk"]
 
 [permissions]
 bors.chalk.review = true
+bors.rust.try = true
```

```suggestion
bors.rust.review = true
bors.rust.try = true
```
needs review permission as well.